admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```diff
@@ -1469,26 +1469,35 @@ $product-page-padding-bottom: 80px !default;
   }
 }
 
-// Combination form page (in modal)
 .combination-page {
-  .combination-images-selector {
-    display: flex;
-    flex-direction: row;
-
+  .combination-images-selector:not(.disabled) {
     .combination-image-choice {
-      margin: 0.35rem;
       cursor: pointer;
-      border: solid transparent 3px;
 
       &:hover,
       &.selected {
         border: solid $primary 3px;
       }
+    }
+  }
+
+  .combination-images-selector {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+
+    .combination-image-choice {
+      margin: 0.35rem;
+      border: solid transparent 3px;
 
       .form-check-radio {
         display: none;
       }
     }
+
+    .combination-images-selector > .combination-image-choice {
+      flex: 0 0 0;
```

Comment:
Why can't you integrate this one above? along with the other `.combination-image-choice` rules? No need to split this yes?